bases/shared/instagoric-server/server.js

```diff
@@ -143,14 +146,22 @@ const getNetworkConfig = async () => {
   ap.peers[0] = ap.peers[0].replace(
     'validator-primary.instagoric.svc.cluster.local',
     svc.get('validator-primary-ext') ||
-      'validator-primary.instagoric.svc.cluster.local',
+      `${podname}.instagoric.svc.cluster.local`,
+  );
+  ap.peers[0] = ap.peers[0].replace(
+    'fb86a0993c694c981a28fa1ebd1fd692f345348b', `${NODE_ID}`,
```

Comment:
Why is this necessary and hardcoded?
```suggestion
    'fb86a0993c694c981a28fa1ebd1fd692f345348b', `${NODE_ID}`,
```

Reply:
Okay, so it's a hardcoded `network_config.json` used in instagoric.  Thanks for explaining it.

Reply:
Thank you for the review.